build_projects/dotnet-host-build/PublishTargets.cs

```diff
@@ -137,12 +138,6 @@ public static BuildTargetResult FinalizeBuild(BuildTargetContext c)
                         "opensuse.42.3.x64.version"
                     };
 
-                    BuildTargetResult feedResult = c.BuildContext.RunTarget(nameof(PublishTargets.PublishCoreHostPackagesToFeed));
```

Comment:
Does `PublishCoreHostPackagesToFeed` still get used/called?  If not, can it be removed?